ci/build_wheel_libwholegraph.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright (c) 2024-2025, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_dir="python/libwholegraph"
+
+export SKBUILD_CMAKE_ARGS="-DBUILD_SHARED_LIBS=OFF;-DCMAKE_MESSAGE_LOG_LEVEL=VERBOSE;-DCUDA_STATIC_RUNTIME=ON"
```

Comment:
Why is `BUILD_SHARED_LIBS=OFF` here?

I'd expect it to be set to `ON`, because we're trying to produce a shared library here, not a static one (docs: https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html).

https://github.com/rapidsai/cugraph-gnn/blob/3d67322d679c8c08e7bd973279db378dff483058/cpp/CMakeLists.txt#L37

Reply:
I think just cargo-culted in from elsewhere, I can probably set this to `ON` and then remove the changes to the `cpp/CMakeLists.txt` file.  I'll try that locally and push up the results!

Reply:
Ok, pushed up and I'll keep an eye on the logs